### PR TITLE
PP-916: Add Recommendation to do a Test 5 Days after Exposure

### PR DESCRIPTION
**Pull request by gallmann-ubique**

### CODE REVIEW

File: app/src/main/res/layout/card_what_to_do_leitfaden.xml [hunk 1]
```diff
@@ -141,6 +141,82 @@
 					android:orientation="vertical"
 					app:layout_constraintTop_toBottomOf="@id/card_encounters_last_call">
 
+					<androidx.constraintlayout.widget.ConstraintLayout
```
Comment (buenzli-ubique) [inline app/src/main/res/layout/card_what_to_do_leitfaden.xml:144]: The two XML layouts seem to have a lot of duplicate layout parts. Could be shared with a common included layout.

File: app/src/main/java/ch/admin/bag/dp3t/reports/ReportsFragment.java [hunk 2]
```diff
@@ -214,6 +223,52 @@ private void deleteNotifications(TracingStatusInterface tracingStatusInterface)
...
+		DayDate oldestExposure = null; // This includes only exposures that are newer than MIN_EXPOSURE_AGE_TO_DO_A_TEST
+
+		for (ExposureDay exposureDay : exposureDays) {
+			if (exposureDay.getExposedDate().addDays(MIN_EXPOSURE_AGE_TO_DO_A_TEST).isBeforeOrEquals(today) &&
+					!exposureDay.getExposedDate().addDays(MAX_EXPOSURE_AGE_TO_DO_A_TEST).isBefore(today)) {
+				testCountdownTextView.setText(R.string.meldungen_detail_free_test_now);
+				return;
+			}
+			if (!exposureDay.getExposedDate().addDays(MIN_EXPOSURE_AGE_TO_DO_A_TEST).isBeforeOrEquals(today) &&
+					exposureDay.getExposedDate().isBeforeOrEquals(today)) {
```
Comment (buenzli-ubique) [inline app/src/main/java/ch/admin/bag/dp3t/reports/ReportsFragment.java:244]: Can an `exposureDay` ever be from the future?

File: app/src/main/res/layout/card_what_to_do_leitfaden.xml (hunk 1 above)
Comment (gallmann-ubique) [reply app/src/main/res/layout/card_what_to_do_leitfaden.xml]: Yes! I have merged this duplicate layout code.

File: app/src/main/java/ch/admin/bag/dp3t/reports/ReportsFragment.java (hunk 2 above)
Comment (gallmann-ubique) [reply app/src/main/java/ch/admin/bag/dp3t/reports/ReportsFragment.java]: No, I have removed this unnecessary check.

**Review by buenzli-ubique — APPROVED** [review]

**Pull request merged**